engine/classes/ElggGroup.php

```diff
@@ -236,78 +219,6 @@ public function leave(\ElggUser $user) {
 	}
 
 	/**
-	 * Load the \ElggGroup data from the database
-	 *
-	 * @param mixed $guid GUID of an \ElggGroup entity or database row from entity table
-	 *
-	 * @return bool
-	 */
-	protected function load($guid) {
```

Comment:
We should probably add `preload_metadata` to ege* to make this smoother. If set, we just join the metadata table and load the values right away.